controllers/mediaController.js

```diff
@@ -0,0 +1,49 @@
+import * as MediaModel from '../models/MediaModel';
+import * as PostModel from '../models/PostModel';
+import getUserByToken from '../utils/getUserFromToken';
+
+const logger = require('../utils/logger')('logController');
+
+const getPosts = async (req, res) => {
+  logger.log('debug', 'getPosts: %j', req.body);
+  const posts = await PostModel.getRandomPosts();
+  res.status(200).send({ payload: !posts ? [] : posts });
```

Comment:
By default it's return null { "payload": null } I don't like this because it should to be consistent 
if we return array so it should be empty in nonthing found